build/wix/scope.wxs

```diff
@@ -43,12 +43,7 @@
         </Control>
         <Control Id="Next" Type="PushButton" Default="yes" Text="!(loc.WixUINext)"
           X="236" Y="243" Width="56" Height="17">
-          <Publish Order="1" Property="MSIINSTALLPERUSER" Value="{}">MSIINSTALLPERUSER = 0</Publish>
-          <Publish Order="2" Property="ALLUSERS" Value="{}">MSIINSTALLPERUSER</Publish>
-          <Publish Order="3" Property="ALLUSERS" Value="1">
-            <![CDATA[MSIINSTALLPERUSER <> 1]]>
-          </Publish>
-          <Publish Order="4" Event="NewDialog" Value="RDVerifyReadyDlg">1</Publish>
+          <Publish Order="1" Event="NewDialog" Value="RDVerifyReadyDlg">1</Publish>
```

Comment:
Does it still need an order property, even if it's a single item in the control?